modules/core/include/opencv2/core/hal/intrin_wasm.hpp

```diff
@@ -2885,18 +2898,28 @@ inline v_int16x8 v_rshr_pack(const v_int32x4& a, const v_int32x4& b)
 template<int n>
 inline v_uint32x4 v_rshr_pack(const v_uint64x2& a, const v_uint64x2& b)
 {
+    #ifdef __wasm_unimplemented_simd128__
```

Comment:
indent: no space at the beginning of the line?

Reply:
Fixed